sympy/printing/latex.py

```diff
@@ -207,7 +207,7 @@ def __init__(self, settings=None):
 
     def parenthesize(self, item, level, strict=False):
         prec_val = precedence_traditional(item)
-        if (prec_val < level) or ((not strict) and prec_val <= level):
+        if (prec_val <= level) or ((not strict) and prec_val <= level):
```

Comment:
An alternative fix would be setting `strict=False` in line 738.  The check of  `(not strict)` in line 210 is now obsolete and can be removed, maybe? 

An alternative approach would be a function `_needs_derivative_brackets(self, expr):`  similar to those in ll. 240ff 

I am not sure, which the proper way to go is, though....

Reply:
This commit is old one. 
I had committed with new changes but doesn't made changes in this PR.